src/Extension/Tasks/AcceptanceTestingTask.yaml

```diff
@@ -4,9 +4,15 @@ short_description: 'Run acceptance tests'
 help: ~
 stage: build
 version: 0.1.0
-command: 'php %sdk_dir%/vendor/bin/codecept run acceptance -o "paths: output: %reportDir%" --json=codeception.report.json'
+command: 'php %sdk_dir%/vendor/bin/codecept run %suites% -o "paths: output: %reportDir%" --json=codeception.report.json'
```

Comment:
I think in this case it makes sense to change id and description of the test too and also check all usages of the task